src/blocks/store/s3.clj

```diff
@@ -221,9 +223,12 @@
         (object->block client bucket prefix stats)
         ; Otherwise, upload block to S3.
         (let [object-key (id->key prefix (:id block))
-              ; TODO: support function which allows you to mutate the ObjectMetadata before it is sent in the putObject call?
               metadata (doto (ObjectMetadata.)
                          (.setContentLength (:size block)))]
+          (when set-sse-flag?
+            (.setSSEAlgorithm metadata ObjectMetadata/AES_256_SERVER_SIDE_ENCRYPTION))
```

Comment:
Instead of a boolean flag, I'd prefer that the library has a mapping from keywords to these algorithm constants. Specifying a keyword not in this map should be an error. Since this is currently the only value, it'd start as:
```clojure
(def sse-algorithms
  "Map of server-side encryption algorithm keywords to string constants."
  {:aes-256 ObjectMetadata/AES_256_SERVER_SIDE_ENCRYPTION})
```